### DISCUSSION TITLE
no settings/wallabag.lua

### DISCUSSION THREAD

**Discussion by pyromaker** [koreader/koreader#9981] [Q&A]
Hello,

I'm trying to [activate](https://github.com/koreader/koreader/wiki/Wallabag) the wallabag service in my device (kobo libra2) but it seems that the settings/wallabag.lua file doesn't exist in my device.

Any help?

**Comment by Frenzie** [comment]
It will only exist if you first set the configuration through the GUI.

**Reply by pyromaker** [reply]
Oh I see, thanks.

But unfortunately I didn't managed to make it work.
I create an API, copy the client ID, client secret, username and password in the wallabag.lua file, restart the koreader and still can't connect to the server (the server url I leave it empty). Also I selected the default folder

Am I missing something?

**Reply by Frenzie** [reply]
It kind of sounds like you might be missing the server URL? Heh.

**Reply by pyromaker** [reply]
Oops, I thought server url must be empty? I thought I read it somewhere in the instructions. :D
And the server url is just the wallabag site? I can't find any url in my profile page

**Reply by Frenzie** [reply]
Generally, yes.
